code/__HELPERS/global_lists.dm

```diff
@@ -53,6 +53,10 @@
 		GLOB.materials_list[D.id] = D
 
 	GLOB.emote_list = init_emote_list()
+	//Skillcapes
+	for(var/path in typecacheof(/obj/item/clothing/neck/skillcape))
```

Comment:
Am I reading this right? You're making a skillcape in nullspace just to store it in a GLOB? Why don't you juse store the path?

Reply:
yes

Reply:
perhaps you should like, make a datum item instead of spawning a full item in nullspace

Reply:
a datum item?